ansible/roles/test/files/ptftests/vxlan-decap.py

```diff
@@ -382,7 +446,10 @@ def checkVxlan(self, acc_port, net_port, test):
         self.dataplane.flush()
         for i in xrange(self.nr):
             testutils.send_packet(self, net_port, packet)
-        nr_rcvd = testutils.count_matched_packets(self, inpacket, acc_port, timeout=0.5)
+        if not wu:
+            nr_rcvd = count_matched_packets_helper(self, inpacket, self.nr, acc_port, timeout=20)
+        else:
+            nr_rcvd = 0
```

Comment:
you don't use this function in the warmup. Should we remove this function?

Reply:
Sure. This logic is not necessary.  But I tend to keep it here until we confirm it's OK to remove checkVxlan from warmup.

Reply:
Updated. Please help to review. Thanks!